README.md

```diff
@@ -167,6 +167,18 @@ Type: `String` (optional)
 
 Brand Title is the brand of your report, e.g. Smoke Tests Report, Acceptance Test Report etc as per your need. If not passed, it will be displayed as "Cucumberjs Report"
 
+#### `timeInNanoseconds`
+Type: `Boolean`
+Default: `false`
```

Comment:
Cucumber's duration is changed back to `nanoSeconds` by default in recent release. Here is the [changelog](https://github.com/cucumber/cucumber-js/blob/master/CHANGELOG.md#400-2017-01-24). I'd suggest to provide option for MicroSeconds and let nanoSeconds by default.